include/CppUTestExt/MockNamedValue.h

```diff
@@ -167,18 +199,28 @@ class MockNamedValueList
  */
 
 struct MockNamedValueComparatorRepositoryNode;
-class MockNamedValueComparatorRepository
+struct MockNamedValueCopierRepositoryNode;
+class MockNamedValueHandlerRepository
 {
-    MockNamedValueComparatorRepositoryNode* head_;
+private:
+    MockNamedValueComparatorRepositoryNode* comparatorsHead_;
+    MockNamedValueCopierRepositoryNode* copiersHead_;
+
 public:
```

Comment:
@basvodde why not have two different lists like this? Especially seeing that we do not -- as you pointed out -- need comparators and copiers together? Then, when we need them, we just look into the appropriate list to start with.